userdocs/contents/overlays.rst

```diff
@@ -178,9 +178,9 @@ To function properly, ``wwctl image syncuser`` (or the ``--syncuser`` option
 during ``import``, ``exec``, ``shell``, or ``build``) must have also been run on
 the image to synchronize its user and group IDs with those of the server.
 
-If a ``PasswordlessRoot`` tag is set to "true", the overlay will also insert a
+If a ``PasswordlessRoot`` tag is set to "true", by uncommenting the top line of /etc/passwd on the provisioned compute node, the overlay will also insert a
```

Comment:
This is not done by uncommenting anything; it is inserted at the top of the file.

https://github.com/warewulf/warewulf/blob/main/overlays/syncuser/rootfs/etc/passwd.ww#L2-L4